tf-modules/cumulus/variables.tf

```diff
@@ -125,11 +125,13 @@ variable "system_bucket" {
 variable "tea_external_api_endpoint" {
   description = "Thin Egress App external endpoint URL"
   type        = string
+  default     = null
```

Comment:
We don't use these for Cumulus Distribution. We could comment/delete them instead of making them optional like this.

Reply:
We can't delete them from the `cumulus` module if we want to support both TEA and cumulus distribution simultaneously, which it seems like we do? Because we're not forcing an upgrade?

Reply:
I'm saying that we have to keep these variables and make them optional like this if we want to keep supporting TEA and new Cognito auth

Reply:
Right. I think this is the best way to go. My worse idea above was leaving them as-is (not optional) and adding a note to the Cognito deploy instructions to delete them if you use Cognito. Which seems more confusing.

Reply:
Resolving this. Seems like we're on the same page and there aren't changes since this thread.